docs/src/main/sphinx/functions/conditional.rst

```diff
@@ -56,15 +56,9 @@ returned if it exists, otherwise null is returned. Example::
 IF
 --
 
-The ``IF`` function is actually a language construct
-that is equivalent to the following ``CASE`` expression:
-
-.. code-block:: text
-
-    CASE
-        WHEN condition THEN true_value
-        [ ELSE false_value ]
-    END
+The standard SQL ``IF`` expression has two forms, one supplying only a
+``true_value`` and the other supplying both a ``true_value`` and a
+``false_value``:
 
 .. function:: if(condition, true_value)
```

Comment:
do we need to add false_value as optional here somehow?

Reply:
it's below